lib/plist-transform.js

```diff
@@ -31,9 +32,29 @@ module.exports = (dir, pkg) => {
 		.then(content => {
 			const data = fix(plist.parse(content));
 			data.version = pkg.version || '';
-			data.description = pkg.description || '';
-			data.webaddress = pkg.homepage || pkg.author.url || '';
-			data.createdby = pkg.author.name || '';
+			data.description = data.description || pkg.description || '';
```

Comment:
Surely there must be a more elegant way to prevent overriding existing keys than this? CoffeeScript has `?=` for example…